tests/test_other.py

```diff
@@ -8131,25 +8082,6 @@ def test_wasm_targets_side_module(self):
         assert not x.endswith('.js'), 'we should not emit js when making a wasm side module: ' + x
       self.assertIn(b'dylink', open(target, 'rb').read())
 
-  def test_wasm_backend_lto(self):
-    if not self.is_wasm_backend():
-      self.skipTest('not using wasm backend')
-    # test codegen in lto mode, and compare to normal (wasm object) mode
-    for args in [[], ['-O1'], ['-O2'], ['-O3'], ['-Os'], ['-Oz']]:
-      print(args)
-      print('wasm in object')
-      run_process([PYTHON, EMCC, path_from_root('tests', 'hello_libcxx.cpp')] + args + ['-c', '-o', 'a.o'])
-      assert Building.is_wasm('a.o') and not Building.is_bitcode('a.o')
-      print('bitcode in object')
-      run_process([PYTHON, EMCC, path_from_root('tests', 'hello_libcxx.cpp')] + args + ['-c', '-o', 'a.o', '-s', 'WASM_OBJECT_FILES=0'])
-      assert not Building.is_wasm('a.o') and Building.is_bitcode('a.o')
-      print('build bitcode object')
-      run_process([PYTHON, EMCC, 'a.o'] + args + ['-s', 'WASM_OBJECT_FILES=0'])
-      self.assertContained('hello, world!', run_js('a.out.js'))
-      print('build with bitcode')
-      run_process([PYTHON, EMCC, path_from_root('tests', 'hello_libcxx.cpp')] + args + ['-s', 'WASM_OBJECT_FILES=0'])
-      self.assertContained('hello, world!', run_js('a.out.js'))
```

Comment:
Why did this test end up moving?     It doesn't look like it should be writing to the cache.  Do we cache bitcode libraries and object file library separately?   If so we should we build the bitcode ones as part of the embuiilder phase too, no?

Reply:
Yes, we have a separate dir for bitcode and object files in the cache. And embuilder doesn't currently have a way to build both, so it just builds the default object files now. So this is the one test which builds the LTO libraries.

Building the LTO ones by default seems costly in terms of build time, but yeah, maybe that's the best thing to avoid confusion.

Reply:
You mean the build time on the circleCI bots?   Slowing down the time it takes to run embuilder?  It would slow down only the `build-upstream` build step right?   Maybe double the time this step takes.  I think this is reasonable, and it won't effect normal users or developers local build times.

Reply:
Just the time it takes to run embuilder, yeah. But normal users may be running embuilder too, and this is a significant slowdown. I think it may be the right way to go, but I'd suggest discussing it in a dedicated issue, I opened https://github.com/emscripten-core/emscripten/issues/8128

Reply:
Moved this test back after landing the `--lto` flag for embuilder.